include/oneapi/dpl/pstl/parallel_backend_tbb.h

```diff
@@ -92,10 +92,12 @@ class __parallel_for_body
 // wrapper over tbb::parallel_for
 template <class _ExecutionPolicy, class _Index, class _Fp>
 void
-__parallel_for(oneapi::dpl::__internal::__tbb_backend_tag, _ExecutionPolicy&&, _Index __first, _Index __last, _Fp __f)
+__parallel_for(oneapi::dpl::__internal::__tbb_backend_tag, _ExecutionPolicy&&, _Index __first, _Index __last, _Fp __f,
+               std::size_t __grainsize = 1)
```

Comment:
I agree that adding a grainsize here is good.  We need a separate PR to look at tuning this for different patterns.  Histogram could use a larger grainsize.

Reply:
Please leave a comment that `1` matches the default grainsize value of `tbb::blocked_range` according to the specification.
Ideally, I would use 0 as the default argument here (because it is meaningless as a grainsize), which would have to be reinitialized to e.g. ``tbb::blocked_range<_Index>{__first,__last}.grainsize()`. But that seems an overkill, since the default grainsize is explicitly specified for oneTBB.

Reply:
A comment was added.